examples/pump-app/cc13x2x7_26x2x7/main/AppTask.cpp

```diff
@@ -192,8 +212,12 @@ int AppTask::Init()
 
     ConfigurationMgr().LogDeviceConfig();
 
+#if defined(CHIP_DEVICE_CONFIG_ENABLE_OTA_REQUESTOR)
+    InitializeOTARequestor();
```

Comment:
Again, this is racing against the already-started event loop.  Needs to be synchronized with it.